tests/broker-engine/services-and-bulk-stmt.robot

```diff
@@ -296,14 +297,16 @@ RLCode
 
     ${INITIAL_SCRIPT_CONTENT}    Catenate
     ...    function init(params)
-    ...    broker_log:set_parameters('/tmp/toto.log', 2)
+    ...    broker_log:set_parameters('/tmp/test4.log', 2)
     ...    end
     ...
     ...    function write(d)
     ...    broker_log:info(0, "toto")
     ...    return true
     ...    end
 
+    Broker Config Add Lua Output    central    test-metric    ${SCRIPTS}test-metric.lua
+
```

Comment:
here you declare the use of a script named 'test-metric.lua', why do you create toto.lua at line 311?
This doesn't look coherent.